components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -387,7 +387,11 @@ class NewTabPage extends React.Component<Props, State> {
 
   fetchBalance = () => {
     chrome.binance.getAccountBalances((balances: Record<string, string>, success: boolean) => {
-      if (!success) {
+      const hasBalances = Object.keys(balances).length
```

Comment:
This will always be empty (`{}`) in an offline response. Also, we shouldn't set auth to invalid if balances simply just don't come in any way. The `success` parameter is the correct thing to go off of for that.